examples/python/CloudantDF.py

```diff
@@ -45,7 +45,7 @@
 
 df.filter(df.airportName >= 'Moscow').select("_id",'airportName').show()
 df.filter(df._id >= 'CAA').select("_id",'airportName').show()
-df.filter(df._id >= 'CAA').select("_id",'airportName').save("airportcodemapping_df",
+df.filter(df._id >= 'CAA').select("_id",'airportName').write.save("airportcodemapping_df",
         "com.cloudant.spark", bulkSize = "100")
```

Comment:
This statement did not work for me to demonstrate the new behavior. I had to modify to:

`df.filter(df._id >= 'ZZZ').select("_id",'airportName').write.save("airportcodemapping_df",
        "com.cloudant.spark", bulkSize = "100")`

in order to get
 `[WARN] [06/02/2016 17:32:50.954] [Thread-3] [CloudantReadWriteRelation(akka://CloudantSpark-a3f15ff1-f340-43eb-a7c5-891fb21f45bb)] Database airportcodemapping_df: nothing was saved because the number of records was 0!`

The original query with `>= CAA` would still give me the good INFO message, like:

`[INFO] [06/02/2016 17:27:43.275] [Executor task launch worker-0] [JsonStoreDataAccess(akka://CloudantSpark-4a0b1607-c66d-491f-b8d8-e6d4cad61cf1)] Save total 13 with bulkSize 100 in 0s`

What behavior did you want to demonstrate here? Pass with or fail with warning?